src/config/env.ts

```diff
@@ -1,11 +1,11 @@
 import { Env } from '~/types';
 
 export const getEnv = (): Env => {
-  const { VITE_RPC_URL, VITE_PROJECT_ID, VITE_ALCHEMY_KEY } = import.meta.env;
+  const { NEXT_PUBLIC_RPC_URL, NEXT_PUBLIC_PROJECT_ID, NEXT_PUBLIC_ALCHEMY_KEY } = process.env;
 
   return {
-    RPC_URL: VITE_RPC_URL,
-    PROJECT_ID: VITE_PROJECT_ID,
-    ALCHEMY_KEY: VITE_ALCHEMY_KEY,
+    RPC_URL: NEXT_PUBLIC_RPC_URL || '',
```

Comment:
@0xArdy @0xtiti  maybe it is better to leave it as undefined and erroring in this case

Reply:
fixed with erroring https://github.com/defi-wonderland/web3-react-boilerplate/pull/8/commits/a4a80bcb3d05cb213b045a2f40932f8ae160615c But its causing issues on github workflow build